iceoryx_utils/include/iceoryx_utils/internal/concurrent/lockfree_queue/cyclic_index.inl

```diff
@@ -81,5 +87,11 @@ bool CyclicIndex<CycleLength, ValueType>::isOneCycleBehind(const CyclicIndex& ot
     }
     return (thisCycle + 1 == otherCycle);
 }
+
+template <uint64_t CycleLength, typename ValueType>
+int64_t CyclicIndex<CycleLength, ValueType>::operator-(const CyclicIndex<CycleLength, ValueType>& rhs) const
+{
+    return m_value - rhs.m_value;
+}
 } // namespace concurrent
 } // namespace iox
```

Comment:
Just had a quick look and saw a couple of file are missing a newline at the very end!
Beside this one it's basically all new files in this PR. Maybe enable the newline option in your IDE?!

Reply:
Will change.